product_supplierinfo_standard_price/models/product_supplierinfo.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (C) 2022 - Today: GRAP (http://www.grap.coop)
+# @author: Quentin DUPONT (quentin.dupont@grap.coop)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
+
+
+class SupplierInfo(models.Model):
+    _inherit = "product.supplierinfo"
+
+    product_standard_price = fields.Float(
+        string="Product actual standard price",
+        compute="_compute_product_standard_price",
```

Comment:
```suggestion
        related="product_tmpl_id.standard_price",
```
could work. 
(and so, we could remove  the _compute_product_standard_price function ? )
don't you think ?